.github/workflows/tests.yml

```diff
@@ -11,8 +11,8 @@ on:
   workflow_dispatch:
   
 env:
-  JAVA_VERSION: '11'
-  JAVA_DISTRIBUTION: 'zulu' #This is the default on v1 of the action for 1.8
+  JAVA_VERSION: '17'
+  JAVA_DISTRIBUTION: 'adopt'
```

Comment:
Why are you changing from Zulu to Adopt?

We use zulu in deployed applications and usually prefer to use the same version in ci as in production.